README.md

```diff
@@ -140,6 +140,44 @@ above; although it is conceivable that such a composite project may define its
 own services, it is unlikely and ill-advised because no one likes blurred lines
 in architectural diagrams. Just look at the Leaning Tower of Pisa.
 
+#### Additional Uberjars
+
+Some projects might have a use case for automatically fetching an versioned
+jar from an external repository that needs to be placed into the package and be
+available after installation.
+
+The `:additional-uberjars` setting can be set to a list of project coordinates
+that will be resolved and have uberjars built from them. For example:
+
+```clojure
+{:lein-ezbake {
+  :additional-uberjars [[puppetlabs/puppetserver "2.7.2"]]
+...}}
+```
+
+This would result in a puppetserver uberjar being built, and the jar being placed in the
+same directory as your project's own uberjar after installation.
+
+The filename of the built uberjar is determined by what value `:uberjar-name`
+is set to in its `project.clj`
+
+ezbake will attempt to resolve the coordinates of these external uberjars using
+the repositories specified in the `:repositories` key of your project.clj
+
+#### cli-defaults.sh
+`cli-defaults.sh` is intended to be a file from which default shell script
+variables can be defined by a project, and be loaded by ezbake's `cli-app`
+script. That is, the script which is run when commands like
+`service my-project start` are called.
+
+This differs from the `default` files, which also get loaded by `cli-app`, in
+that a project has no control over what goes into `default`. `cli-defaults.sh`
+is under the control of your project.
+
+For `cli-defaults.sh` to be used, it simply needs to exist at
```

Comment:
Hmmm, I tried this out with pe-puppetserver as my composite project and it didn't work the way I had been thinking it would.  I tried putting different versions of `./resources/ext/cli_defaults/cli-defaults.sh.erb` file into a pe-puppet-server-extensions and a pe-puppetserver jar and the ezbake stage from pe-puppetserver picked up the pe-puppet-server-extensions one, not the pe-puppetserver one.  I saw the same thing for other resources, e.g., for a cli app I stored under ext/cli.

Maybe ezbake doesn't automatically look at the resources in the immediate project's jar - at least unless you self reference the immediate project as a dependency [like we do in OSS puppetserver](https://github.com/puppetlabs/puppetserver/blob/2.7.2/project.clj#L143)?